scripts/data/summeval_fr/create_data.py

```diff
@@ -0,0 +1,58 @@
+import datasets
+import json
+import requests
+
+from huggingface_hub import create_repo, upload_file
+
+ORGANIZATION = "lyon-nlp/"
+REPO_NAME = "summarization-summeval-fr-p2p"
+SAVE_PATH = "test.json"
+
+summeval = datasets.load_dataset("mteb/summeval")["test"]
+headers = {
+    'Authorization': 'DeepL-Auth-Key ****',
```

Comment:
I think it's better to put an environment varibale for authentication keys. Add a comment at the top to explain that the variable **DEEPL_AUTH_KEY** should be configured for running the script : 

```
import os 

API_KEY= os.environ.get('DEEPL_AUTH_KEY')

headers = {
    'Authorization': f"DeepL-Auth-Key {API_KEY}"
```